packages/mdl-radio/event-bus.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Class to coordinate between between components without strict dependencies.
+ */
+export default class EventBus {
```

Comment:
Also, using the adjacent sibling combinator approach would help us get rid of the initial state syncing we have to do within the `MDLRadio` constructor below.